scully/utils/defaultAction.ts

```diff
@@ -46,7 +46,7 @@ export const generateAll = async (localBaseFilter = baseFilter) => {
     performanceIds.add('Discovery');
     log('Pull in data to create additional routes.');
     const handledRoutes = await addOptionalRoutes(
-      unhandledRoutes.filter((r: string) => r && r.startsWith(localBaseFilter))
+      unhandledRoutes.filter((r: string) => typeof r === 'string' && r.startsWith(localBaseFilter))
```

Comment:
I am still not sure if this check is needed at all. Maybe it makes sense to just omit it?

Reply:
It protects to mistakes in the Scully.config file. so, yes the check is needed. We have no control over the data that is coming from the guess parser, and neither on the config file. If a route is not a string, its pretty safe to ignore it.

Reply:
That makes sense, thank you.